apps/backend/src/evaluations/evaluations.controller.ts

```diff
@@ -80,13 +85,17 @@ export class EvaluationsController {
     );
   }
 
+  @UseGuards(APIKeyOrJwtAuthGuard)
   @Post()
   @UseInterceptors(FileInterceptor('data'), CreateEvaluationInterceptor)
   async create(
     @Body() createEvaluationDto: CreateEvaluationDto,
     @UploadedFile() data: Express.Multer.File,
     @Request() request: {user: User}
   ): Promise<EvaluationDto> {
+    if (!request.user) {
```

Comment:
Why was this check added? What behavior changed between API or JWT guard and simple JWT guard?

Reply:
The passport-jwt library seems to throw UnauthorizedException by itself, however the lookup for API keys is handled on our side. The handling for this seems better suited for APIKeyOrJwtAuthGuard though.